bgcval2/analysis_timeseries.py

```diff
@@ -88,6 +89,29 @@ def listModelDataFiles(jobID, filekey, datafolder, annual):
     return model_files
 
 
+def list_input_files(files_path, dictionary, paths):
```

Comment:
don't name a functional arg `dictionary` - it's too general and it's a builtin too - give it a more descriptive name pls

Reply:
Done.